test/NormalizePragmasSpec.hs

```diff
@@ -0,0 +1,38 @@
+module NormalizePragmasSpec where
+
+import           Test.Hspec
+
+import           NormalizePragmas
+
+spec :: Spec
+spec = do
+  describe "sortPragmas" $ do
+    it "sort pragmas" $ do
+      let input = [
+              "{-# LANGUAGE B #-}"
+            , "{-# LANGUAGE Ab #-}"
+            , "{-# OPTIONS_GHC Abc #-}"
+            , "{-# LANGUAGE A #-}"
+            ]
+          output = [
+              "{-# LANGUAGE A #-}"
+            , "{-# LANGUAGE Ab #-}"
+            , "{-# LANGUAGE B #-}"
+            , "{-# OPTIONS_GHC Abc #-}"
+            ]
```

Comment:
I wouldn't care about `OPTIONS_GHC` at all. They are so seldom that I'm perfectly fine with formatting them manually.